public/v4/apps/checkmk.yml

```diff
@@ -0,0 +1,33 @@
+captainVersion: 4
+services:
+    '$$cap_appname':
+        image: checkmk/check-mk-raw:$$cap_version
+        environment:
+            TZ: '$$cap_tz'
+            PHP_TZ: '$$cap_tz'
+        volumes:
+            - '$$cap_appname-data:/omd/sites'
+            - /etc/localtime:/etc/localtime:ro
```

Comment:
`:ro` is not read by CapRover. Can be removed.

Reply:
removed it